machAeroTutorials/overset_theory.rst

```diff
@@ -45,18 +73,36 @@ If there are overlapping meshes, it basically uses the smaller cells, and the co
 .. figure:: images/overset_IHC.png
     :align: center
 
-    Before (left) and after IHC (right).
+    The original mesh (left) and only the compute cells after IHC (right).
 ..
     src: overset_guide.pdf page 7
 
+The ``iBlank`` array indicates the function of each cell.
+ADflow saves this array in the volume CGNS files.
+
+.. figure:: images/overset_guide_8.jpg
+    :align: center
+
+    The resulting iBlank values after the IHC process for the background and near-field meshes.
+..
+    src: overset_guide.pdf page 8
+
+In the figure above, the red cells represent the compute cells in each mesh.
+The green cells are the interpolate cells, which bring in information from the overlapping compute cells.
+The dark blue are the flood-seeds; these are the cells in the background mesh that overlap with a surface in another mesh (the circle in this case).
+The light blue cells are the flooded cells; these were connected to the flood-seeds so they are marked as flooded.
+The flood-seed and flooded cells have no function in the actual solution, The yellow cells represent the blanked cells.
+These have no function in the actual solution; however, 2 layers of blank or interpolate cells are required to *stop the flooding*.
```

Comment:
his looks super relevant, do you think we should turn this into a note/warning or is it "common knowledge"?

Reply:
Converted this to a note.